Makefile

```diff
@@ -0,0 +1,26 @@
+# Go parameters
+GOCMD=go
```

Comment:
Is it common on go project to have a Makefile?

Reply:
I knew someone would pick on the GOCMD =) 
regarding the makefile I believe it helps both for developing, manual testing and CI. cc @rafie